.github/workflows/ci_tests.yml

```
@@ -74,36 +75,46 @@ jobs:
           ninjaVersion: "^1.11.1"
       - name: Print installed softwares
         run: |
-          clang++ --version
-          g++ --version
+          ${{ matrix.compiler.cpp }} --version || ${CXX} --version || echo ignored
+          ${{ matrix.compiler.c }} --version || ${CC} --version || echo ignored
```

```suggestion
          ${{ matrix.compiler.cpp }} --version || echo ignored
          ${{ matrix.compiler.c }} --version || echo ignored
```

I don't think `CXX` and `CC` environment variable are not defined throughout the action.
It sucks that we have to do this for macos though.